code/android-core-library/src/main/java/com/adobe/marketing/mobile/internal/eventhub/ExtensionContainer.kt

```diff
@@ -299,6 +300,7 @@ internal class ExtensionContainer constructor(
         )
     }
 
+    @Deprecated("Deprecated in Java")
```

Comment:
nit: `Deprecated in Java` may be a bit confusing for implementers using other JVM languages. Suggest switching to `Deprecated`